### PR TITLE
VR: Add the VR code to all builds. Remove IsVRBuild calls from the renderer.

### PR DESCRIPTION
**Pull request by hrydgard**
First step towards refactoring things so that the VR code is always built on desktop and can be selectively enabled.

Adds all the VR code to all the build systems. Generally, I don't want the build systems to include/exclude code, each CPP file should itself be responsible for checking PPSSPP_PLATFORM and so on. 

Will definitely need a few buildfixes, so marking as draft.

### CODE REVIEW

**Comment by lvonasek** [comment]
I see you started it big 😉. I will keep my hands off for a few days to prevent conflicts.

Just a few notes:
* `PPSSPP_PLATFORM(ANDROID)` define is not a good way to avoid GL/XR code.
* I would really love to keep VR*.* files independ of PPSSPP to be able to use it in other VR projects

If you need a help, I can offer you a pair programming this Friday afternoon (CET).

**Comment by hrydgard** [comment]
Yeah I guess I can avoid ppsspp_config.h for now, I'll change the ifdefs.

I don't know how realistic it is to make it a reusable library in the long run though, I feel that might restrict the API and organization of the code a little much... We'll see.

**Comment by lvonasek** [comment]
Thank you, it somehow surprised me that you started this multiplatform work. I have honestly some doubts that it is a good way to go.

**Comment by hrydgard** [comment]
I generally like to compile all code for all platforms, as far as it's possible, even if some ends up completely unused on some platforms. Ends up checking more code with more compilers on CI, ending up with less breakage overall.

**Comment by hrydgard** [comment]
OK, I think it should be more to your liking now. Tested and works.

File: Common/VR/VRInput.cpp [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #include "VRInput.h"
 #include <string.h>
-#include <sys/time.h>
+
+#include <Common/TimeUtil.h>
```
Comment (unknownbrackets) [inline Common/VR/VRInput.cpp:4]: Oops, `<>`?

-[Unknown]

**Review by lvonasek — CHANGES_REQUESTED** [review]
It looks better but I see still a several `include "Common/..."` in `VR*.*`

If you don't have GL on the target platform you should use `ifdef XR_USE_GRAPHICS_API_OPENGL_ES` instead of `ifdef ANDROID`.

If you don't have XR on the target platform you should use `ifdef OPENXR` instead of `ifdef ANDROID`.

**Pull request merged**